src/java/org/apache/cassandra/repair/messages/RepairOption.java

```diff
@@ -313,21 +307,23 @@ else if (ranges.isEmpty())
     private final boolean repairPaxos;
     private final boolean paxosOnly;
 
-    private final boolean accordRepair;
+    private final boolean accordOnly;
+    private final boolean isConsensusMigration;
```

Comment:
I don't really agree. Understanding what consistency repair should perform a given operation at seems is a bit specialized for someone reading the code for repair or consensus migration tool.

Reply:
Totally reasonable. I think a comment somewhere explaining that consensus migration wants to be highly available so it can occur at QUORUM while repair wants to ensure visibility of data before the repair started at all replicas.

Really migration being highly available is just icing on the cake since we can theoretically stay migrating indefinitely and it is fine. It just works that way because it's additional work to add all which you have now done.

I think it's worth keeping just because we don't really know the downsides of being in migration yet.